src/types.rs

```diff
@@ -449,6 +449,21 @@ pub enum VectorialStrafingConstraints {
         /// The highest yaw angle of the range in degrees.
         to: f32,
     },
+    /// Constrains the player yaw to look at the given point.
+    LookAt {
+        #[cfg_attr(
+            feature = "proptest1",
+            proptest(strategy = "any::<u32>().prop_map(NonZeroU32::new)")
+        )]
+        /// If set, the coordinates will be added to the origin of an entity with this index.
```

Comment:
nooo, i deleted that comment, please undo, that was wrong

Reply:
Xdd